nbs/src/tbats.ipynb

```diff
@@ -26,11 +26,11 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.        for k in range(len(combinations)): 
</pre><p>These can be extracted like:</p><pre class="ql-syntax" spellcheck="false">for boxcox, (trend, damped_trend), arma_errors in combinations:
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Got it. </p>